docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -515,6 +515,10 @@ following properties:
 * - `iceberg.rest-catalog.nested-namespace-enabled`
   - Support querying objects under nested namespace.
     Defaults to `false`.
+* - `iceberg.rest-catalog.view-endpoints-enabled`
+  - Enable view endpoints. Defaults to `true`.
+* - `iceberg.rest-catalog.sigv4-enabled`
```

Comment:
Could you also document `iceberg.rest-catalog.signing-name` config property?

Reply:
Done :)